src/main/java/ch/njol/skript/util/slot/EquipmentSlot.java

```diff
@@ -34,170 +35,242 @@
  * Represents equipment slot of an entity.
  */
 public class EquipmentSlot extends SlotWithIndex {
-	
+
+	/**
+	 * Enum to handle the setting of the equipment slot.
+	 * 
+	 * @deprecated The multiple naming of the same enums is confusing.
+	 * Spigot now has an exact enum for handling the exact same Equipment Slots.
+	 * We're planning to remove this enum and only use Spigot's EquipmentSlot.
+	 */
+	@Deprecated
+	@ScheduledForRemoval
+	// Developers note: These set and get methods should be implemented in a switch inside the EquipmentSlot#setItem method or make the enum private.
 	public enum EquipSlot {
```

Comment:
What's going on? Why is equipment slot stuff being changed in a harvest event PR?

Reply:
It's needed for this event, so it's mainly the EquipmentSlot changes

Reply:
I believe this would be better off in it's own PR where we can properly deprecate this and update existing methods. I don't believe we should be deprecating in 1 PR and not update all current internal uses. If you wanna say EquipmentSlot changes is the main target of this PR then harvest event should be moved into it's own PR where it's within scope of it.

Based off your comment about it, you *or the team* intend to remove the enum in favor of spigots if that's truly the plan then we should update all current instances to spigot's in the pr deprecating this. So we don't have more work to do when we finally remove it in a year or so.

I do note believe this is within scope of your original plan of doing harvest event for this pr.